healsparse/healSparseMap.py

```diff
@@ -1195,6 +1242,10 @@ def valid_pixels(self):
             valid_pixel_inds, = np.where(self._sparse_map[self._primary] != self._sentinel)
         elif self._is_wide_mask:
             valid_pixel_inds, = np.where(np.any(self._sparse_map != self._sentinel, axis=1))
+        elif self._is_bit_mask:
```

Comment:
Yup.  I'm thinking about new APIs for `valid_pixels_chunks` (or something with a better name; ideas welcome) that will give a nice iterator.  I also know how to make the coverage generator a view rather than a copy which will make it easier to use.